### PR TITLE
Extend Admission Controllers 101

### PR DESCRIPTION
**Pull request by chipzoller**
Preview link: https://deploy-preview-1155--kyverno.netlify.app/docs/introduction/admission-controllers/

## Related issue #

#857 

## Proposed Changes

Greatly extends the Admission Controllers 101 docs.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by chipzoller** [comment]
No other takers? Ok.

File: content/en/docs/Introduction/admission-controllers.md [hunk 1]
```diff
@@ -1,115 +1,324 @@
...
 
-- **Configuration management:** Admission controllers allow you to validate the configuration of the objects running in the cluster and prevent any obvious misconfigurations from hitting your cluster. Admission controllers can be useful in detecting and fixing images deployed without semantic tags, such as by automatically adding resource limits or validating resource limits, ensuring reasonable labels are added to pods, or ensuring image references used in production deployments are not using the latest tags, or tags with a -dev suffix.
+- **Fine-grained RBAC:** Traditional RBAC (Role-Based Access Control) defines roles and permissions to perform specific actions within a Kubernetes cluster, but it lacks the granularity needed for certain scenarios. Dynamic admission controllers address this gap by allowing administrators to impose detailed conditions on top of standard RBAC policies. For example, the ability to place restrictions on who may delete a Namespace with a certain label is only possible through dynamic admission control.
```
Comment (realshuting) [inline content/en/docs/Introduction/admission-controllers.md:43]: One of the common questions I've seen is that the user wants to restrict RBAC permissions such as get/list/watch through an admission controller, which is not doable. You may want to mention that.

File: content/en/docs/Introduction/admission-controllers.md [hunk 2]
```diff
@@ -1,115 +1,324 @@
...
+
+The number and type of these field defaultings and mutations can vary based on the resource type, the admission controllers enabled in the cluster, and other factors. Since a dynamic admission controller has no way of distinguishing between what you, as a person, submitted to the API server versus what it received it is important to take these factors into consideration when authoring and troubleshooting policy.
+
+Understanding the sequence of events leading up to dynamic admission controllers being invoked is important, but so is how they are invoked when the time arrives.
+
+During the dynamic mutating admission phase, the webhooks are called serially where one is called followed by the next and so on. They are not called at random but in [lexical order](https://en.wikipedia.org/wiki/Lexicographic_order). For example, if there were two `MutatingWebhookConfigurations`, one named `alpha` and the other `bravo`, the `alpha` webhook will be called first and all responses returned before `bravo` is called. The ordering cannot be influenced aside from changing the name of the MutatingWebhookConfiguration resource itself.
```
Comment (realshuting) [inline content/en/docs/Introduction/admission-controllers.md:258]: Nice! I was looking for this while reading.

**Review by realshuting — COMMENTED** [review]
Late to the party but this looks really good!

File: content/en/docs/Introduction/admission-controllers.md (hunk 1 above)
Comment (chipzoller) [reply content/en/docs/Introduction/admission-controllers.md:43]: This is already present below: https://main.kyverno.io/docs/introduction/admission-controllers/#limitations

**Pull request merged**